<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>QNumberSettingsWidget</class>
 <widget class="QWidget" name="QNumberSettingsWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>450</width>
    <height>64</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QHBoxLayout" name="horizontalLayout">
   <item>
    <widget class="QLabel" name="label_2">
     <property name="text">
      <string>Precision:</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QSpinBox" name="precision_SpnBox">
     <property name="minimum">
      <number>1</number>
     </property>
     <property name="value">
      <number>2</number>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QLabel" name="label_3">
     <property name="text">
      <string>Interval:</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QSpinBox" name="minVal_SpnBox">
     <property name="minimum">
      <number>0</number>
     </property>
     <property name="value">
      <number>10</number>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QSpinBox" name="maxVal_SpnBox">
     <property name="value">
      <number>20</number>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
